src/azul/project/hca/transformers.py

```diff
@@ -252,8 +255,10 @@ def visit(self, entity: api.Entity) -> None:
         elif isinstance(entity, api.Organoid):
             self.organoids[entity.document_id] = entity
         elif isinstance(entity, api.Process):
+            visitable_protocols = (api.SequencingProtocol, api.LibraryPreparationProtocol,
```

Comment:
As previously requested, we wrap all or none. If you dislike the wrap-all-or-none rule, consider that I essentially picked the rule that is least ambiguous. Once I picked a rule, we need to follow it, for consistency's sake, however arbitrary and subjectively ugly it may seem to you.

And please inline the variable, it doesn't add anything.

Reply:
Sorry, I didn't understand the "wrap all or none" rule. Corrected the layout.